### PR TITLE
Updated and revised

### PR DESCRIPTION
**Pull request by LezaiYa**

### CODE REVIEW

File: Modules/CustomRolesHelper.cs [hunk 1]
```diff
@@ -1061,8 +1062,7 @@ public static bool CheckAddonConfilct(CustomRoles role, PlayerControl pc)
                     || pc.Is(CustomRoles.Vindicator) 
                     || pc.Is(CustomRoles.TicketsStealer)
                     || pc.Is(CustomRoles.Pickpocket) 
-                    || pc.Is(CustomRoles.Glitch)
-                    || pc.Is(CustomRoles.Dictator))
+                    || pc.Is(CustomRoles.Glitch))
```
Comment (Tommy-XL) [inline Modules/CustomRolesHelper.cs:1065]: Why Dictator was removed?

Comment (LezaiYa) [reply Modules/CustomRolesHelper.cs:1065]: Dictator's skip is also counted and VoidBallot won't influence dictator's skill.

Comment (Tommy-XL) [reply Modules/CustomRolesHelper.cs:1065]: Then this Add-on is simply useless for him, although he could get a more useful add-on

**Comment by Loonie-Toons** [comment]
Has conflicts, pls resolve :P

File: Modules/CustomRolesHelper.cs (hunk 1 above)
Comment (Loonie-Toons) [reply Modules/CustomRolesHelper.cs:1061]: Dictator should not be able to be Void Ballot.

**Comment by NikoCat233** [comment]
> Disallow Dictator from getting Void Ballot lol. It can't vote normally, so it should not get the addon.

Ok pls follow lonnie's idea
Although it wont glitch if dictator get voidballot

**Comment by LezaiYa** [comment]
> Disallow Dictator from getting Void Ballot lol. It can't vote normally, so it should not get the addon.

Ok, I know, I'll make the changes when I can play on the computer while I'm on vacation, I'll close this pull request first

**Pull request closed without merging**